content/pt-br/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Implantar e acessar o Kubernetes Dashboard
+description: >-
+  Implementa a interface web do usuário (Kubernetes Dashboard) e acessá-lo.
+content_type: concept
+weight: 10
+card:
+  name: tasks
+  weight: 30
+  title: Usar a interface de usuário do Dashboard web
+---
+
+
+<!-- overview -->
+
+Dashboard é uma interface de usuário Kubernetes baseada na Web.
+Você pode usar o dashboard para implantar aplicações contêineres para um cluster Kubernetes,
+solucione problemas de seu aplicativo conteinerizado e gerencie os recursos do cluster.
+Você pode usar o dashboard para obter uma visão geral dos aplicações em execução no seu cluster,
+bem como para criar ou modificar recursos individuais do Kubernetes
+(como Deploys, tarefas, daemonsets, etc).
```

Comment:
Here we don't need translate *Jobs*, because is related to a kubernetes resource: Jobs